_includes/head.html

```diff
@@ -22,8 +22,11 @@
   <link rel="alternate" type="application/atom+xml" title="{{ site.title | escape }} (it)" href="{{ site.baseurl }}/it/news/feed.atom">
   <link rel="alternate" type="application/atom+xml" title="{{ site.title | escape }} (en)" href="{{ site.baseurl }}/en/news/feed.atom">
 
-  <link href="{{ site.baseurl }}/assets/fonts.css?{{ buildTime }}" rel="stylesheet">
-  <link href="{{ site.baseurl }}/assets/main.css?{{ buildTime }}" rel="stylesheet">
+  {% if jekyll.environment == "production" %}
+  <link href="{{ site.baseurl }}/dist/main.css?{{ buildTime }}" rel="stylesheet">
+  {% else %}
+  <link href="//localhost:8080/main.css" rel="stylesheet">
+  {% endif %}
```

Comment:
This will breaks netlify build as it uses `make local`. See here https://deploy-preview-647--developers-italia.netlify.app/

Reply:
Yeah, I mentioned it in the description. I'm using
`make build-swagger && make download-data && make include-npm-deps && JEKYLL_ENV=production jekyll build && npm run build` [on my Netlify build](https://adoring-lalande-cb9e63.netlify.app/).

We'll need to add the environment variable and `npm run build` to the current one as well.

(I think in the future we can consolidate all that as `make build`)

Reply:
sorry, my bad. :+1: